app/repo.rb

```diff
@@ -35,39 +35,45 @@ def production_hosted_on_aws?
   end
 
   def machine_class
-    app_data["machine_class"] || aws_puppet_class
+    repo_data["machine_class"] || aws_puppet_class
   end
 
   def production_hosted_on
-    app_data["production_hosted_on"]
+    repo_data["production_hosted_on"]
+  end
+
+  def is_app?
+    !production_hosted_on.nil?
   end
 
   def hosting_name
     Hosts::HOSTERS.fetch(production_hosted_on)
   end
 
   def html_url
-    "https://docs.publishing.service.gov.uk/apps/#{app_name}.html"
+    "https://docs.publishing.service.gov.uk/repos/#{app_name}.html"
   end
 
   def retired?
-    app_data["retired"]
+    repo_data["retired"]
   end
 
   def private_repo?
-    app_data["private_repo"]
+    repo_data["private_repo"]
   end
 
   def page_title
     if retired?
       "Application: #{app_name} (retired)"
```

Comment:
This looks like things turn back into apps when they get retired regardless of whether they were beforehand?

Reply:
Good spot. Fixed (see final commit) 👍